uid-pages/page-admin-profile-list/src/adminProfileListBonitaV1.json

```diff
@@ -1265,6 +1265,215 @@
                         }
                       ]
                     ]
+                  },
+                  {
+                    "type" : "container",
+                    "dimension" : {
+                      "xs" : 12,
+                      "sm" : 3,
+                      "md" : 3,
+                      "lg" : 3
+                    },
+                    "propertyValues" : {
+                      "cssClasses" : {
+                        "type" : "constant",
+                        "value" : ""
+                      },
+                      "hidden" : {
+                        "type" : "expression",
+                        "value" : "featuresListCtrl.isFeatureAvailable(\"WEB_PROFILE_PO\")"
```

Comment:
the mapping can be done on provided profiles as well. no need for feature check in this case